taccsite_cms/templates/guides/rsync_data_transfer.html

```diff
@@ -0,0 +1,510 @@
+      </p>
+      <p>
+        If you wish to learn more about <code>rsync</code> and how to synchronize your file transfer, you can do so
+        <a target="_blank" href="https://man7.org/linux/man-pages/man1/rsync.1.html">the online <code>man</code> page for <code>rsync</code></a>
+        or follow the file transfer section of the user guide for the relevant storage system:
+      </p>
+      <ul>
+        <li>
+          <a target="_blank" href="https://portal.tacc.utexas.edu/user-guides/stampede2#transferring-rsync">Stampede2 User Guide: Transferring Files Using <code>rsync</code></a>
+        </li>
+        <li>
+          <a target="_blank" href="https://frontera-portal.tacc.utexas.edu/user-guide/files/#transferring-files-with-rsync">Frontera User Guide: Transferring Files with <code>rsync</code></a>
+        </li>
+        <li>
+          <a target="_blank" href="https://portal.tacc.utexas.edu/user-guides">TACC User Guides (for each system)</a>
+          <small>(select the relevant system for this portal, then search "<code>rsync</code>")</small>
+        </li>
+      </ul>
+
+      <h4 id="cyberduck_transfer">Transfer Data with Cyberduck</h4>
+
+      <p>
+        <a target="_blank" href="https://cyberduck.io/">Cyberduck</a> is a free graphical tool for data transfer and is an alternative to using the command line. With a drag-and-drop interface, it is easy to transfer a file from your local system to the remote secure system. You can use
+        <a target="_blank" href="https://cyberduck.io/">Cyberduck</a>
+        for Windows or macOS.
+      </p>
+      <h5 id="cy_windows">For Windows</h5>
+
+      <p>
+        <a target="_blank" href="https://cyberduck.io/download/">Download and install Cyberduck for Windows</a>
+        on your local machine.
+      </p>
+
+      <p>
+        Once installed, click “Open Connection” in the top left corner of your Cyberduck window.
+      </p>
+
+      <figure id="figure2">
+        <img src="https://a2cps-staging.tacc.utexas.edu/media/filer_public_thumbnails/filer_public/a9/9f/a99fa278-ae07-4b5e-9219-f9db3c2c89d5/picture2.png__1170x0_q85_subsampling-2_upscale.png" />
+        <figcaption>
+          Figure 2. Windows Cyberduck and “Open Connection” setup screen
+        </figcaption>
+      </figure>
+      <p>
+        To setup a connection, type in the server name
+        <kbd>host</kbd> (ex. <code>secure.corral.tacc.utexas.edu</code>). Add your TACC username and password in the spaces provided. If the "More Options" area is not shown, click the small triangle button to expand the window; this will allow you to enter the path to your transfer directory <kbd>/transfer/directory/path</kbd> (ex.
+        <code>/corral-secure/projects/A2CPS/submissions/tacc</code>) so that when Cyberduck opens the connection you will immediately be in your individualized transfer directory on the system. Click the "Connect" button to open your connection.
```

Comment:
SUGGESTION: Remove the example path, it has already been established earlier in the document and makes this section harder to comprehend.